docs/editor/artificial-intelligence.md

```diff
@@ -0,0 +1,90 @@
+---
+Order: 7
+Area: editor
+TOCTitle: Refactoring
+ContentId:
+PageTitle: Code smarter with artifical intelligence
+DateApproved: 1/27/2023
+MetaDescription: Code smarter with AI-powered suggestions from GitHub Copilot.
+---
+# Code smarter with artificial intelligence
```

Comment:
Just AI is probably better, to avoid the mouthful overloaded long term.

Reply:
Updated to "Code smarter with AI"